arbnode/inbox_reader.go

```diff
@@ -533,7 +533,7 @@ func (r *InboxReader) GetLastReadBlockAndBatchCount() (uint64, uint64) {
 	return r.lastReadBlock, r.lastReadBatchCount
 }
 
-// >0 - last batchcount seen in run() - only written after lastReadBatchCount updated
+// GetLastSeenBatchCount >0 - last batchcount seen in run() - only written after lastReadBatchCount updated
 // 0 - no batch seen, error
 func (r *InboxReader) GetLastSeenBatchCount() uint64 {
```

Comment:
The comment here doesn't really document what it does, just the return values. Perhaps we could prefix it with:
```
GetLastSeenBatchCount returns how many sequencer batches the inbox reader has read in from L1
```